src/View/SSViewer_BasicIteratorSupport.php

```diff
@@ -2,6 +2,8 @@
 
 namespace SilverStripe\View;
 
+use SilverStripe\Dev\Deprecation;
```

Comment:
The markdown template docs need to be updated. I would probably talk about both syntaxes, eg. "As of CMS 4.7 we introduce IsFirst & IsLast. The original syntax of First and Last still work, but will be deprecated in the future."